src/scores/continuous/standard_impl.py

```diff
@@ -240,9 +242,9 @@ def additive_bias(
     fcst: XarrayLike,
     obs: XarrayLike,
     *,
-    reduce_dims: XarrayLike = None,
-    preserve_dims: XarrayLike = None,
-    weights: XarrayLike = None,
+    reduce_dims: Optional[XarrayLike] = None,
```

Comment:
See last review, 
TLDR;
Should this be `FlexibleDimensionTypes`?

Reply:
Yes, thanks, good pickup

Reply:
FYI, fixes pushed into the final MR so I can confirm nothing new is revealed and all things are finally resolved.